source/faucet/main.d

```diff
@@ -268,7 +268,7 @@ public class Faucet : FaucetAPI
         // AA keys are addresses
         auto builder = TxBuilder(
             secret_keys.byKey().drop(uniform(0, secret_keys.length, rndGen)).front());
-        utxo_rng.each!(kv => builder.attach(kv.value.output, kv.key));
```

Comment:
This is just a style difference, it doesn't actually change anything

Reply:
IMO it is less error prone as we have some static validation checks in the `Range of tuples` passed to the `TxBuilder` `attach` function which is specifically for handling a `range` of `tuple(Output, Hash)`.